main.py

```
@@ -2,47 +2,51 @@
 # Integrates with UMDR MCP stack for full runtime capabilities
 
 from agents.executor import execute_task
-from agents.mutator import mutate_protocol, analyze_all_protocols
+from agents.mutator import mutate_protocol
```

Pylint (3.3.7)

[error] Cannot import 'agents.mutator' due to 'unterminated string literal (detected at line 27) (agents.mutator, line 27)'

(E0001)